docs/storage/bestman-install.md

```diff
@@ -0,0 +1,500 @@
+
+        :::shell
+        # Number of files to keep
+        BESTMAN_EVENT_LOG_COUNT=10
+        # Size of each file in bytes
+        BESTMAN_EVENT_LOG_SIZE=20971520
+
+    The optimal value for these depends on usage of the service.
+
+- Create a directory under a much bigger partition and have a symlink from `/var/log/bestman2` to that directory.
+- Leave the default settings, but have your own custom script that cleans these files according to your needs.
+
+Starting Services
+=================
+
+1. `fetch-crl`
+
+    For RHEL 6:
+
+        :::console
+        [root@client ~] # /usr/sbin/fetch-crl   # This fetches the CRLs 
+        [root@client ~] # /sbin/service fetch-crl-boot start
+        [root@client ~] # /sbin/service fetch-crl-cron start
+
+    For RHEL 7:
+
+        :::console
+        [root@client ~] # /usr/sbin/fetch-crl   # This fetches the CRLs 
+        [root@client ~] # systemctl start fetch-crl-boot
+        [root@client ~] # systemctl start fetch-crl-cron
+
+2. GridFTP
+
+        :::console
+        [root@client ~] # service globus-gridftp-server start
+
+3. Bestman
+
+        :::console
+        [root@client ~] # service bestman2 start
+
+    To start Bestman automatically at boot time
+
+        :::console
+        [root@client ~] # chkconfig bestman2 on
+
+4. Gratia transfer and storage probes
+
+        :::console
+        [root@client ~] # service gratia-gridftp-transfer start
+
+Stopping Services
+=================
+
+1. fetch-crl
+
+    For RHEL 6:
+
+        :::console
+        [root@client ~] # /usr/sbin/fetch-crl   # This fetches the CRLs
+        [root@client ~] # /sbin/service fetch-crl-boot start
+        [root@client ~] # /sbin/service fetch-crl-cron start
+
+    For RHEL 7:
+
+        :::console
+        [root@client ~] # /usr/sbin/fetch-crl   # This fetches the CRLs
+        [root@client ~] # systemctl start fetch-crl-boot
+        [root@client ~] # systemctl start fetch-crl-cron
+
+2. GridFTP
+
+        :::console
+        [root@client ~] # service globus-gridftp-server stop
+
+3. Bestman
+
+        :::console
+        [root@client ~] # service bestman2 stop
+
+4. Gratia transfer and storage probes
+
+        :::console
+        [root@client ~] # service gratia-gridftp-transfer start
+
+
+Validation of Service Operation
+===============================
+
+Once you have your SE setup and configured, there are several ways to monitor your installation. Refer to the following pages for more information:
+
+* [BeStMan SRM Tester](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/SrmTester).
+* [RSV](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallRSV) which includes SRM probes as well.
+
+You can also self-test to verify your installation with an SRM client such as `gfal-copy`.
+
+Troubleshooting
+===============
+
+Useful Configuration and Log Files
+----------------------------------
```

Comment:
Perhaps combine this with the "Useful Configuration and Log Files" section below